Qt 6.8

benma opened the pull request:
Before asking for reviews, here is a check list of the most common things you might need to consider:
- [ ] updating the Changelog
- [ ] writing unit tests
- [ ] checking if your changes affect other coins or tokens in unintended ways
- [ ] testing on multiple environments (Qt, Android, ...)
- [ ] having an AI review your changes

benma commented:
The two failing tests are unrelated.

Beerosagos reviewed (changes requested):
I think there are still a few occurrences of the old Qt version around:

```
scripts/github-ci.sh:35:    export LDFLAGS="-L~/Qt/6.2.4/macos/lib"
scripts/github-ci.sh:36:    export CPPFLAGS="-I~/Qt/6.2.4/macos/include"
docs/BUILD.md:42:aqt list-qt mac desktop --arch 6.2.4
docs/BUILD.md:43:aqt install-qt mac desktop 6.2.4 --modules qtpositioning qtserialport qtwebchannel qtwebengine --outputdir ~/Qt
docs/BUILD.md:49:export PATH="$PATH:$HOME/Qt/6.2.4/macos/bin"
docs/BUILD.md:50:export PATH="$PATH:$HOME/Qt/6.2.4/macos/libexec"
README.md:53:- [Qt](https://www.qt.io) version 6.2.4
```

benma commented on frontends/qt/compile_windows.bat:
```diff
@@ -1,14 +1,15 @@
-COPY "%VCToolsRedistDir%\x64\Microsoft.VC142.CRT\vccorlib140.dll" windows\
-COPY "%VCToolsRedistDir%\x64\Microsoft.VC142.CRT\vcruntime140.dll" windows\
-COPY "%VCToolsRedistDir%\x64\Microsoft.VC142.CRT\vcruntime140_1.dll" windows\
+dir "%VCToolsRedistDir%\x64\
```
Thanks for the catch, it's a debug leftover.

benma commented:
> I think there are still a few occurrences of the old Qt version around:
> 
> ```
> scripts/github-ci.sh:35:    export LDFLAGS="-L~/Qt/6.2.4/macos/lib"
> scripts/github-ci.sh:36:    export CPPFLAGS="-I~/Qt/6.2.4/macos/include"
> docs/BUILD.md:42:aqt list-qt mac desktop --arch 6.2.4
> docs/BUILD.md:43:aqt install-qt mac desktop 6.2.4 --modules qtpositioning qtserialport qtwebchannel qtwebengine --outputdir ~/Qt
> docs/BUILD.md:49:export PATH="$PATH:$HOME/Qt/6.2.4/macos/bin"
> docs/BUILD.md:50:export PATH="$PATH:$HOME/Qt/6.2.4/macos/libexec"
> README.md:53:- [Qt](https://www.qt.io) version 6.2.4
> ```

Fixed

I removed the LDFLAGS/CPPFLAGS as CI passed with the wrong version, so it seems it's not needed at all.

Beerosagos reviewed (approved):
utACK!
